pkgs/tools/security/ghidra/doc/tests/eclipse-01.nix

```diff
@@ -0,0 +1,8 @@
+# Expected result:
+# An Eclipse instance should start, with the GhidraDev
+# plugin attached as a debugger to a headless Ghidra
+# that is built with an out-of-tree plugin. The debugger
+# should stop in a breakpoint in the plugin source code
+# and display the appropriate source location.
+
+#TODO not yet implemented
```

Comment:
I'd say either implement this right away or do not check in this stubbed test for now.

Reply:
Giving up on this for the moment, possible starting points:
https://testing.bredex.de/test-machine-setup.html https://www.eclipse.org/rcptt/
I have no idea how to work any of this stuff.